census

```diff
@@ -0,0 +1,621 @@
+#!/usr/bin/python
+
+'''Evaluate apache logs from send-census.
+
+This maintains a local SQLite database for the historic data and the current
+state, and can produce text and chart outputs.
+'''
+
+__author__ = 'Matt Zimmerman <matt.zimmerman@canonical.com>, Martin Pitt <martin.pitt@canonical.com>'
+__copyright__ = '(C) 2010 Canonical Ltd.'
+__license__ = 'GPL v2 or later'
+
+import random
+import re
+import sys
+import time
+import datetime
+import os
+import os.path
+import urlparse
+import operator
+import optparse
+import sqlite3 as dbapi2
+import subprocess
+import tempfile
+import gzip
+import GeoIP
+import csv
+import fileinput
```

Comment:
Alphabetize the imports and mark any that are non default in a base install with comments for easier maintenance:
`import foobar # Requires python-foobar package`